packages/system/src/Location/Components/Toolbar.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { FC, useEffect, useRef } from 'react';
```

Comment:
since this is only used in the list view, maybe should live in that folder?

Reply:
Good point - ta